tirtha_bk/static/css/styles.css

```diff
@@ -1035,10 +1087,10 @@ progress::-webkit-progress-value {
     height: clamp(0.8rem, 1.1rem, 1.5dvw);
 
     accent-color: var(--agree-text-col);
-    box-shadow:     ;
+    box-shadow: ;
```

Comment:
Was this property empty originally?

Reply:
i think that was empty! Although I have set the property to "none" now.

Reply:
Okay, it was somehow empty.

Reply:
Set this to `none`.